dependencies/graknlabs/artifacts.bzl

```diff
@@ -27,5 +27,6 @@ def graknlabs_grakn_core_artifacts():
         artifact_name = "grakn-core-server-{platform}-{version}.tar.gz",
         tag_source = deployment_private["artifact.release"],
         commit_source = deployment_private["artifact.snapshot"],
-        commit = "6a6d6d834f96895dc56e8e107ff1d603ae0d2846",
+        # TODO: this artifact has DEBUG log level and netty 4.1.51
+        commit = "3344528e74dbe1a8567c00dc9574830f2a24fa96",
```

Comment:
This needs to be updated once https://github.com/graknlabs/grakn-2.0/pull/98 is merged and has deployed successfully